cmd/builder/README.md

```diff
@@ -57,11 +57,34 @@ $ /tmp/dist/otelcol-custom --config=/tmp/otelcol.yaml
 
 ## Installation
 
-There are two supported ways to install the builder: via the official releases (recommended) and through `go install`.
+There are three supported ways to install the builder:
+1. Via official release Docker images (recommended)
+2. Via official release binaries (recommended)
+3. Through `go install` (not recommended)
 
-### Official releases 
+### Official release Docker image
 
-This is the recommended installation method. Download the binary for your respective platform under the ["Releases"](https://github.com/open-telemetry/opentelemetry-collector/releases?q=builder) page.
+You will find the official docker images at [DockerHub](https://hub.docker.com/r/otel/opentelemetry-collector-builder). 
+
+Pull the image via tagged version number (e.g. v0.110.0) or 'latest.' You may also specify platform, although Docker will handle this automatically as it is a multi-platform build.
+
+```
+docker pull otel/opentelemetry-collector-builder:latest
+```
+
+The included builder configuration file/manifest should be replaced by mounting a file from your local filesystem to the docker container; the default location is `/build/builder-config.yaml`. If you mount a file at a different location inside the container, your `builder.config.yaml` must be specified as a command line argument to ocb. Additionally, the output folder must also be mounted from your local system to the docker container. This output directory must be specified in your `builder-config.yaml` file as it cannot be set via the command-line arguments.
+
+Assuming you are running this image in your working directory, have a `builder-config.yaml` file located in this folder, the `dist.output_path` item inside your `builder-config.yaml` is set to `./otelcol-dev`, and you wish to output the binary/go module files to a folder named "output," the command would look as follows:
+
+```
+docker run -v "$(pwd)/builder-config.yaml:/build/builder-config.yaml" -v "$(pwd)/output:/build/otelcol-dev" otel/opentelemetry-collector-builder:latest
+```
+
+Additional arguments may be passed to ocb on the command line as specified below, but if you wish to do this, you must make sure to pass the `--config` argument, as this is specified as an additional `CMD`, not an entrypoint. 
+
+### Official release binaries 
+
+This is also a recommended installation method. Download the binary for your respective platform under the ["Releases"](https://github.com/open-telemetry/opentelemetry-collector/releases?q=builder) page.
```

Comment:
Please make sure to update this link as in https://github.com/open-telemetry/opentelemetry-collector/pull/11260/files.